### PR TITLE
refs #3811 alerts messages cleanup and minor refactoring

### PR DESCRIPTION
**Pull request by ohadlevy**

### CODE REVIEW

**Comment by ohadlevy** [comment]
@abenari could you review please?

File: app/views/config_templates/_form.html.erb [hunk 1]
```diff
@@ -70,11 +70,11 @@
 
 
       <div class="tab-pane" id="template_associations">
-        <p id="snippet_message" class="alert alert-message" <%= display? !@config_template.snippet %> ><%= _("Not relevant for snippet") %></p>
+        <p id="snippet_message" class="alert alert-info" <%= display? !@config_template.snippet %> ><%= _("Not relevant for snippet") %></p>
 
         <div id="association" <%= display? @config_template.snippet %> >
-          <div class="alert alert-message alert-success">
-            <a class="close" href="#" data-dismiss="alert">&times;</a>
+          <div class="alert alert-success dismissable">
```
Comment (abenari) [inline app/views/config_templates/_form.html.erb]: shouldn't it be alert-dismissible?

**Comment by ohadlevy** [comment]
@abenari fixed

**Comment by ohadlevy** [comment]
@abenari ping?

**Comment by abenari** [comment]
merged as ede5687c5a58b7b10fb6ec6ae6ba3951801073bf

**Pull request closed without merging**